bower.json

```diff
@@ -22,6 +22,11 @@
     "test",
     "tests"
   ],
+  "dependencies": {
+    "angular": "~1.4.4",
+    "angular-sanitize": "~1.4.4",
```

Comment:
I understand that. This comment is for this specific line that makes this module depend on Angular Sanitize.  Same for jQuery. The only dependency is Angular.